README.md

```diff
@@ -20,27 +20,18 @@
 
 
 ##### Code
-
+- Grab the das-apply-service configuration json file from [das-employer-config](https://github.com/SkillsFundingAgency/das-employer-config/blob/master/das-roatp-oversight/SFA.DAS.RoatpOversight.json)
```

Comment:
Should that be "das-roatp-oversight", not apply service?

Reply:
yes